Q2ShawnCampbell.md

```diff
@@ -0,0 +1,148 @@
+# Quiz #2
+
+## Instructions
+
+1. Fork this repo
+2. Clone your fork
+3. Fill in your answers by writing in the appropriate area, or placing an 'x' in
+the square brackets (for multiple-choice questions).
+4. Add/Commit/Push your changes to Github.
+5. Open a pull request.
+
+## CSS
+
+### Question #1
+
+Describe the purpose of a clearfix in CSS, and give an example of how to do it.
+
+Your Answer:
+```The clearfix is a CSS method used to make it easier to format a group of
+images.
+```
+
```

Comment:
While we did use it to format a group of images in the in-class example, the most important thing is that we can use clearfix to clear floated elements inside a container, and force that container to actually contain them.

Take a look at this blog post for a refresher on all things float and clear: https://css-tricks.com/all-about-floats/